newrelic-agent/src/main/java/com/newrelic/agent/Agent.java

```diff
@@ -9,12 +9,7 @@
 
 import com.google.common.collect.ImmutableMap;
 import com.newrelic.agent.bridge.AgentBridge;
-import com.newrelic.agent.config.AgentConfig;
-import com.newrelic.agent.config.AgentJarHelper;
-import com.newrelic.agent.config.ConfigService;
-import com.newrelic.agent.config.ConfigServiceFactory;
-import com.newrelic.agent.config.JarResource;
-import com.newrelic.agent.config.JavaVersionUtils;
+import com.newrelic.agent.config.*;
```

Comment:
We don't use * imports. Should be refactored to individual imports.

Reply:
updated